apps/web/pages/api/basenames/frame/assets/registrationFrameImage.png.tsx

```diff
@@ -0,0 +1,174 @@
+import { ImageResponse } from '@vercel/og';
+import { NextRequest } from 'next/server';
+import { namehash } from 'viem';
+import { base } from 'viem/chains';
+import { isDevelopment } from 'apps/web/src/constants';
+import { getBasenamePublicClient } from 'apps/web/src/hooks/useBasenameChain';
+import { openGraphImageHeight, openGraphImageWidth } from 'apps/web/src/utils/opengraphs';
+import { getUserNamePicture, UsernameTextRecordKeys } from 'apps/web/src/utils/usernames';
+import { USERNAME_L2_RESOLVER_ADDRESSES } from 'apps/web/src/addresses/usernames';
+import L2ResolverAbi from 'apps/web/src/abis/L2Resolver';
+import ImageRaw from 'apps/web/src/components/ImageRaw';
+import registrationImageBackground from 'apps/web/public/images/frames/basenames/registration-bg.png';
```

Comment:
Nitpick: would not put this in public folder if it's imported. can be moved close to where it's used.

Reply:
moved closer and it's still working locally